src/actions/helpers/getPlayersHelper.js

```diff
@@ -0,0 +1,22 @@
+import socket from '../../socketHandlers/index';
+
+const SOCKET_EVENT_TYPES = {
+  EMIT: 'getPlayers',
+  ON: 'resultGetPlayers'
+}
+
+const getPlayersPromise = (roomId) => {
+  return new Promise((resolve, reject) => {
+    socket.on(SOCKET_EVENT_TYPES.ON, (data) => {
+      if(data.result.error) {
+        resolve(data.result.error);
```

Comment:
This comment is also to me. 
Instead of `resolve`, we should use `reject`, because this is error pattern.